### DISCUSSION TITLE
Bienvenue !

### DISCUSSION THREAD

**Discussion by sgigou** [sgigou/ibepo#56] [General]
Mettons à profit ce que les développeurs de GitHub nous offrent : bienvenue dans les discussions.

Ici, vous pouvez poser des questions ou émettre des remarques sans avoir à ouvrir de ticket.

J’essaierai de répondre aussi vite que possible. J’attends vos suggestions avec impatience !

À bientôt !

**Comment by japotrad** [comment]
Bonjour. Mon compte Apple est en Thaïlande. Est-ce que vous pourriez rendre votre appli achetable depuis ce pays-là, SVP ?
